templates/R53-hostedzone.yaml

```diff
@@ -1,15 +1,12 @@
 Description: Setup a hosted zone for DNS
 AWSTemplateFormatVersion: 2010-09-09
 Parameters:
-  Department:
-    Description: The department for this resource (i.e. Computational Oncology)
-    Type: String
-  Project:
-    Description: The name of the project that this resource is used for (i.e. Resilience)
-    Type: String
   OwnerEmail:
```

Comment:
OwnerEmail is also used a tag which I don't think we need either.  is there any reason why you are keeping this around?

Reply:
yeah, it's not as well implemented as the CostCenter cost category (there are many more untagged resources), but there's also an effort to create an Owner cost category, and this is one of the tags that cost category is derived from

Reply:
the question is do we need `OwnerEmail` in this template?  can we just pass it in as a cloudformation tag?

Reply:
I can imagine a scenario where a project team uses this template and wants to have a specific team member responsible for DNS, but combing through our current usage this doesn't seem to be the case today - and taking a step back, we would probably want the template isolated to its own cloudformation stack in that case, so I'm going to remove this as well and we can add it back if we need to.